apps/protocol-frontend-e2e/cypress.config.ts

```diff
@@ -11,9 +12,18 @@ module.exports = defineConfig({
   chromeWebSecurity: false,
   e2e: {
     setupNodeEvents(on, config) {
-      return;
+      on('task',{
+        queryDatabase: query => {
+          return queryDB(query)
+        },
+  
+      })
     },
     specPattern: './src/integration/**/*.cy.{js,jsx,ts,tsx}',
     supportFile: './src/support/index.ts',
   },
+  env: {
+    "address" : process.env['ADDRESS'],
+    "COOKIE" : process.env['COOKIE'],
+  },
```

Comment:
Can this env key be removed?

Reply:
I just removed it in all branches. Thanks for the reminder :)